client/landing/stepper/declarative-flow/site-setup-flow.ts

```diff
@@ -164,12 +164,17 @@ const siteSetupFlow: Flow = {
 					if ( ! siteSlug ) {
 						return;
 					}
+
+					// If user clicks "Skip to Dashboard" on goals step,
+					// Default site_intent to "build"
+					const updatedSiteIntent = intent ? intent : SiteIntent.Build;
```

Comment:
Since we're setting a default value for _any_ flow with some falsey value for `intent`, would we inadvertently enable launchpad for situations where it's undesired?